src/test/java/ru/ayaz/UserTest.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class UserTest {
+
+    private User user;
+
+    @Before
+    public final void before(){ user = new User(); }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void passingNull_setName() throws Exception {
```

Comment:
I believe it is the recommended style from the book on TDD ?
I would, though, recommend you to use naming pattern like this for your test methods:
shouldGetZeroWhenTheDatabaseIsEmpty or, if you prefer shouldGetZero_whenTheDatabaseIsEmpty - or something like that. Underscores are usually considered to lie outside of Java world and camelCase is used almost exclusively. Although, if you find a version that is easier to read even with _ in it, I'll agree to it :)